File: app/src/main/java/ai/elimu/content_provider/rest/EmojisService.java
package ai.elimu.content_provider.rest;

import java.util.List;

import ai.elimu.model.v2.gson.content.EmojiGson;
import retrofit2.Call;
import retrofit2.http.GET;

public interface EmojisService {

    @GET("content/emojis")
    Call<List<EmojiGson>> listEmojis();
}

File: app/src/main/java/ai/elimu/content_provider/rest/EmojisService.kt
package ai.elimu.content_provider.rest

import ai.elimu.model.v2.gson.content.EmojiGson
import retrofit2.Call
import retrofit2.http.GET

interface EmojisService {
    @GET("content/emojis")
    fun listEmojis(): Call<List<EmojiGson>>
}
